install_config/install/advanced_install.adoc

```diff
@@ -1972,6 +1972,69 @@ Moving from a single master cluster to multiple masters after installation is
 not supported.
 ====
 
+[discrete]
+[[single-master-multi-node-ai]]
+==== Single Master and Multiple Nodes
+The following table describes an example environment for a single
+xref:../../architecture/infrastructure_components/kubernetes_infrastructure.adoc#master[master] (with etcd installed on the same host)
+and two
+xref:../../architecture/infrastructure_components/kubernetes_infrastructure.adoc#node[nodes]:
+
+[options="header"]
+|===
+
+|Host Name |Infrastructure Component to Install
+
+|*master.example.com*
+|Master, etcd, and node
+
+|*node1.example.com*
+.2+.^|Node
+
+|*node2.example.com*
+|===
+
+You can see these example hosts present in the *[masters]*, *[etcd]*, and
+*[nodes]* sections of the following example inventory file:
+
+.Single Master and Multiple Nodes Inventory File
+----
+# Create an OSEv3 group that contains the masters, nodes, and etcd groups
+[OSEv3:children]
+masters
+nodes
+etcd
+
+# Set variables common for all OSEv3 hosts
+[OSEv3:vars]
+# SSH user, this user should allow ssh based auth without requiring a password
+ansible_ssh_user=root
+
+# If ansible_ssh_user is not root, ansible_become must be set to true
+#ansible_become=true
+
+openshift_deployment_type=openshift-enterprise
```

Comment:
Should we also consider origin like elsewhere.

```
 ifdef::openshift-enterprise[]
 openshift_deployment_type=openshift-enterprise
 endif::[]
 ifdef::openshift-origin[]
 openshift_deployment_type=origin
 endif::[]
```

Reply:
I'll update other examples as well for this one.